src/app/pipes/meter.pipe.spec.ts

```diff
@@ -1,8 +1,34 @@
 import { MeterPipe } from './meter.pipe';
 
 describe('MeterPipe', () => {
+  let pipe: MeterPipe;
+  beforeEach( () => pipe = new MeterPipe());
+
   it('create an instance', () => {
-    const pipe = new MeterPipe();
     expect(pipe).toBeTruthy();
   });
+
+  it('should transform 1324.456 | meter to "1324m"', () => {
```

Comment:
Do you think we should test this every time we use the pipe (with real values instead of test values)? (I don't, but still)

Reply:
Tests are not meant to be run when we use them, but when we commit changes, to quickly check if any past functionalities broke.